banner/index.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta property="og:url" content="http://www.etcher.io" />
+    <meta property="og:type" content="website" />
+    <meta property="og:title" content="Etcher by Resin.io" />
+    <meta property="og:description" content="Burn images to SD cards and USB drives, safe and easy." />
+    <meta property="og:image" content="../static/images/interface.png" />
+    <link rel="shortcut icon" type="image/png" href="../static/images/etcher.ico"/>
```

Comment:
Only a minor nit, but I guess we don't really need all these meta-tags, if this page isn't going to be indexed by web-crawlers etc. ?